doc/contribute/guidelines.rst

```diff
@@ -573,12 +573,14 @@ workflow here:
    you want to open a pull request with.
 
 #. Review the pull request changes, and verify that you are opening a
-   pull request for the appropriate branch. The title and message from your
-   commit message should appear as well.
-
-#. If you're working on a subsystem branch that's not ``main``,
-   you may need to change the intended branch for the pull request
-   here, for example, by changing the base branch from ``main`` to ``net``.
+   pull request for the appropriate branch (if unsure, use `main`). The title
+   and message from your commit message should appear as well.
+
+   .. note:: As more commits are merged upstream, the GitHub PR page will show
```

Comment:
Not sure if this is the best place to add this note. IMHO, it would make more sense if this note is placed after `#. Click on the submit button and your pull request is sent and awaits review.`

Reply:
Oh, definitely! Moved.